### PR TITLE
[admin] Doubles Admin Event Cancellation Time 

### PR DESCRIPTION
**Pull request by Moja10**
Its too hard to click on event cancellation prompt when admin logs are spamming at the speed of sound. I have missed event cancellations because I want to cancel it, but I don't make it on time. Or that I didn't realize because I had the audacity to look away or type for 5 seconds, and then only have a 5 second window to cancel. Doubles the event cancellation timer which doesn't really affect anything on the player's side, just lets admins more easily ward off events that really don't need to be spawned.

# Changelog
🆑 Moja10, Bibby
tweak: The Gods can more easily prevent station shattering events from happening before the consequences are ever known. 
/🆑

### CODE REVIEW

File: code/modules/events/_event.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#define RANDOM_EVENT_ADMIN_INTERVENTION_TIME 10
+#define RANDOM_EVENT_ADMIN_INTERVENTION_TIME 20
```
Comment (Bibby0110) [inline code/modules/events/_event.dm]:
```suggestion
#define RANDOM_EVENT_ADMIN_INTERVENTION_TIME (2 SECONDS)
```

**Review by Bibby0110 — APPROVED** [review]

**Comment by MegaEmpirical** [comment]
Fucking yes. Any chance you can make the text more obvious too? Shit is small and blends in with all the other text going on.

File: code/modules/events/_event.dm [hunk 2]
```diff
@@ -83,7 +83,7 @@
 	triggering = TRUE
 	if(alert_observers)
 		message_admins("Random Event triggering in [RANDOM_EVENT_ADMIN_INTERVENTION_TIME] seconds: [name] (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)")
```
Comment (Bibby0110) [inline code/modules/events/_event.dm]:
```suggestion
		message_admins("Random Event triggering in [RANDOM_EVENT_ADMIN_INTERVENTION_TIME / 10] seconds: [name] (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)")
```

**Review by adamsong — CHANGES_REQUESTED** [review]
Make it red again

**Comment by Moja10** [comment]
This should already be occuring from PR #11076

> Make it red again

I never made it red to begin with you [REDACTED], but sure, I am a generous God.

File: code/modules/events/_event.dm [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#define RANDOM_EVENT_ADMIN_INTERVENTION_TIME 10
+#define RANDOM_EVENT_ADMIN_INTERVENTION_TIME (20 SECONDS)
```
Comment (adamsong) [inline code/modules/events/_event.dm:1]: 200 seconds is a bit excessive

**Review by adamsong — CHANGES_REQUESTED** [review]

File: code/modules/events/_event.dm [hunk 4]
```diff
@@ -82,8 +82,8 @@
 
 	triggering = TRUE
 	if(alert_observers)
-		message_admins("Random Event triggering in [RANDOM_EVENT_ADMIN_INTERVENTION_TIME] seconds: [name] (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)")
-		sleep(RANDOM_EVENT_ADMIN_INTERVENTION_TIME SECONDS)
+		message_admins("<span class='bold'><font color = red>Random Event triggering in [RANDOM_EVENT_ADMIN_INTERVENTION_TIME] seconds: [name] (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)</font color></span>")
```
Comment (adamsong) [inline code/modules/events/_event.dm]: Divide the time by 10

**Review by adamsong — CHANGES_REQUESTED** [review]

**Review by adamsong — APPROVED** [review]

File: code/modules/events/_event.dm [hunk 5]
```diff
@@ -82,8 +82,8 @@
 
 	triggering = TRUE
 	if(alert_observers)
-		message_admins("Random Event triggering in [RANDOM_EVENT_ADMIN_INTERVENTION_TIME] seconds: [name] (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)")
-		sleep(RANDOM_EVENT_ADMIN_INTERVENTION_TIME SECONDS)
+		message_admins("<span class='bold'><font color = red>Random Event triggering in [RANDOM_EVENT_ADMIN_INTERVENTION_TIME/10] seconds: [name] (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)</font color></span>")
```
Comment (Absolucy) [inline code/modules/events/_event.dm]:
```suggestion
		message_admins(span_bold(span_red("Random Event triggering in [RANDOM_EVENT_ADMIN_INTERVENTION_TIME/10] seconds: [name] (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)")))
```

**Review by Absolucy — CHANGES_REQUESTED** [review]

**Comment by ToasterBiome** [comment]
Adam steals the span macros and then forgets to use them

**Pull request merged**